Header header
moveit_msgs/RobotTrajectory trajectory
bool ee_close_at_start
bool is_pick_at_end
bool is_place_at_end
---
bool success
std_msgs/String desc
